plugins/modules/aci_l3out.py

```diff
@@ -346,21 +483,6 @@ def main():
 
     aci.get_existing()
 
-    child_configs = [
-        dict(l3extRsL3DomAtt=dict(attributes=dict(tDn="uni/l3dom-{0}".format(domain)))),
-        dict(l3extRsEctx=dict(attributes=dict(tnFvCtxName=vrf))),
-    ]
-    if l3protocol is not None:
-        for protocol in l3protocol:
-            if protocol == "bgp":
-                child_configs.append(dict(bgpExtP=dict(attributes=dict(descr="", nameAlias=""))))
-            elif protocol == "eigrp":
-                child_configs.append(dict(eigrpExtP=dict(attributes=dict(descr="", nameAlias="", asn=asn))))
-            elif protocol == "ospf":
-                child_configs.append(dict(ospfExtP=dict(attributes=dict(descr="", nameAlias=""))))
-            elif protocol == "pim":
-                child_configs.append(dict(pimExtP=dict(attributes=dict(descr="", nameAlias=""))))
-
     if state == "present":
         aci.payload(
             aci_class="l3extOut",
```

Comment:
should we also add missing attribute "mplsEnabled" while we are working on this module?

Reply:
also in the payload there is dn specified, is this needed?

Reply:
I will add this missing attribute.

you're right, the dn is not needed. I just added the ospf spec and I wanted to wait for your reviews to modify what was originally written.